openapi/SwarmCommon.yaml

```diff
@@ -0,0 +1,214 @@
+openapi: 3.0.0
+info:
+  version: '0.1.0'
+  title: 'Common Data Types'
+  description: |
+    \*****bzzz*****
+
+externalDocs:
+  description: Browse the documentation @ the Swarm Docs
+  url: 'https://swarm-gateways.net/bzz:/d70fb7f255b0f7ab4a0e53c186ca9803636344afde868d8d967167775a682291/'
+
+paths: {}
+components:
+  schemas:
+
+    Address:
+      type: object
+      properties:
+        Address:
+          $ref: '#/components/schemas/SwarmAddress'
+
+    Addresses:
+      type: object
+      properties:
+        overlay:
+          $ref: '#/components/schemas/SwarmAddress'
+        underlay:
+          type: array
+          items:
+            $ref: '#/components/schemas/P2PUnderlay'
+
+     
+    BzzChunksPinned:
+      type: object
+      properties:
+        chunks:
+          type: array
+          items:
+            type: object
+            properties:
+              address:
+                type: string
+              pinCounter:
+                type: integer
+
+    BzzTopology:
+      type: object
+      properties:
+        baseAddr:
+          $ref: '#/components/schemas/SwarmAddress'
+        population:
+          type: integer
+        connected:
+          type: integer
+        timestamp:
+          type: string
+        nnLowWatermark:
+          type: integer
+        depth:
+          type: integer
+        bins:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              population:
+                type: integer
+              connected:
+                type: integer
+              disconnectedPeers:
+                type: object
+              connectedPeers:
+                type: object
+
+    DateTime:
+      type: string
+      format: date-time
+      pattern: '^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{7}\+\d{2}:\d{2})$'
+      example: "2020-06-11T11:26:42.6969797+02:00"
+
+    FileName:
+      type: string
+
+    Hash:
+      type: object
+      properties:
+        hash:
+          $ref: '#/components/schemas/SwarmAddress'
+   
+    MultiAddress:
+      type: string
+
+    Msd5:
```

Comment:
This should be called differently as it is time duration, so a suggestion `Duration`. Also to add a description that it is in Go's time.Duration format.